### PR TITLE
CC-16934 Glue support for Unzer IG.

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

- Epic: https://spryker.atlassian.net/browse/CC-16934

- Ticket for IG: https://spryker.atlassian.net/browse/CC-16965

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by kkicheglovspryker — DISMISSED** [review]

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+{% endinfo_block %}
+
+---
+
+### 2) Set up the configuration
+
+Put all the payment methods available in the shop to  `PaymentsRestApiConfig`:
+
+**src/Pyz/Glue/CheckoutRestApi/CheckoutRestApiConfig.php**
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:49]: file looks wrong

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Create a cart by sending the `POST https://glue.mysprykershop.com/carts`.
+2. Add at least one item to the cart by sending `POST https://glue.mysprykershop.com/{{cart_uuid}}/items`.
+3. Check result by sending the `POST https://glue.mysprykershop.com/checkout-data?include=payment-methods` request.
+
+**Request:**
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:183]:
```suggestion
**Request Example:**
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+{% info_block infoBox %}
+
+Please take care that requests body differs for each Unzer payment method.
+- Property `paymentMethodName` of `payments` has to be replaced by used method (e.g. `Unzer Sofort`, `Unzer Credit Card`, etc.). 
+- Property `paymentResource` of `payments` is not required for `Unzer Sofort`, `Unzer Marketplace Sofort`, `Unzer Bank Transfer`.
+
+{% endinfo_block %}
+
+**Response:**
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:211]:
```suggestion
**Response Example:**
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+      }
+    },
+    ...
+  ]
+}
+```
+
+4. Check result by sending the `POST https://glue.mysprykershop.com/checkout` request.
+
+**Request:**
```
Comment (DmitryLymarenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:261]: same here and below

**Review by DmitryLymarenko — APPROVED** [review]

**Review by AdvaBD — DISMISSED** [review]

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+        }
+      ]
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
+
+Take care that requests body differs for each Unzer payment method.
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:204]:
```suggestion
Ensure that the request body differs for each Unzer payment method.
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+- Property `paymentMethodName` of `payments` has to be replaced by the used method—for example, `Unzer Sofort` or `Unzer Credit Card`.
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:205]:
```suggestion
- Property `paymentMethodName` of `payments` must be replaced by the used method—for example, `Unzer Sofort` or `Unzer Credit Card`. 
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+
+{% info_block infoBox %}
+
+Take care that requests body differs for each Unzer payment method.
+- Property `paymentMethodName` of `payments` has to be replaced by the used method—for example, `Unzer Sofort` or `Unzer Credit Card`. 
+- Property `paymentResource` of `payments` is not required for `Unzer Sofort`, `Unzer Marketplace Sofort`, and `Unzer Bank Transfer`.
+
+{% endinfo_block %}
+
+**Response Example:**
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:210]:
```suggestion
<details><summary markdown='span'>Response example:</summary>
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+      },
+      "links": {
+        "self": "https://glue.mysprykershop.com/payment-methods/1"
+      }
+    },
+    ...
+  ]
+}
+```
+
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:257]:
```suggestion

</details>

```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 9]
```diff
@@ -0,0 +1,313 @@
...
+      ],
+      ...
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
+
+Take care that requests body differs for each Unzer payment method.
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:288]:
```suggestion
Ensure that the request body differs for each Unzer payment method:
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 10]
```diff
@@ -0,0 +1,313 @@
...
+- Property `paymentMethodName` of `payments` has to be replaced by used method (e.g. `Unzer Sofort`, `Unzer Credit Card`, etc.).
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:289]:
```suggestion
- Property `paymentMethodName` of `payments` must be replaced by the used method—for example, `Unzer Sofort` or `Unzer Credit Card`.
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 11]
```diff
@@ -0,0 +1,313 @@
...
+- Property `paymentResource` of `payments` is not required for `Unzer Sofort`, `Unzer Marketplace Sofort`, `Unzer Bank Transfer`.
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:290]:
```suggestion
- Property `paymentResource` of `payments` is not required for `Unzer Sofort`, `Unzer Marketplace Sofort`, and  `Unzer Bank Transfer`.
```

Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:290]:
```suggestion
- Property `paymentResource` of `payments` is not required for `Unzer Sofort`, `Unzer Marketplace Sofort`, and  `Unzer Bank Transfer`.
```

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 12]
```diff
@@ -0,0 +1,313 @@
...
+- Property `paymentSelection` of `payments` has to be replaced by used method (e.g. `unzerSofort`, `unzerCreditCard`, `unzerMarketplaceCreditCard`, etc.).
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:291]:
```suggestion
- Property `paymentSelection` of `payments` has to be replaced by used method—for example, `unzerSofort`, `unzerCreditCard`, or `unzerMarketplaceCreditCard`.
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 13]
```diff
@@ -0,0 +1,313 @@
...
+    {
+        return [
+            new UnzerNotificationResource(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:177]: Add an introductory sentence explaining what we verify—for example, "Verify that the following API requests work:"

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 14]
```diff
@@ -0,0 +1,313 @@
...
+          "amount": "{{amount}}"
+        }
+      ],
+      ...
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:286]: If the steps listed in this infoBox are mandatory, then replace `{% info_block infoBox %}` with  `{% info_block warningBox %}`

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 15]
```diff
@@ -0,0 +1,313 @@
...
+          "paymentMethodName": "Unzer Sofort",
+          "paymentProviderName": "Unzer"
+        }
+      ]
+    }
+  }
+}
+```
+
+{% info_block infoBox %}
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:202]: If the steps listed in this infoBox are mandatory, then replace `{% info_block infoBox %}` with  `{% info_block warningBox %}`

File: docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md [hunk 16]
```diff
@@ -0,0 +1,318 @@
...
+description: Integrate the Glue support in Unzer into your project
+last_updated: Aug 10, 2022
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the *Glue support* to the [Unzer](/docs/pbc/all/payment-service-providers/unzer/unzer.html) into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Glue for Unzer.
```
Comment (VadymSachenko) [inline docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer-glue.md:12]: I think this sentence is even better.
```suggestion
Follow the steps below to install the Unzer Glue API feature core.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**